backend/account/views.py

```diff
@@ -37,9 +37,10 @@ def profile(request, user_pk):
     """
     prof = get_object_or_404(Profile, pk=user_pk)
     if request.method == 'GET':
-        if not request.user.is_authenticated:
-            return HttpResponse(status=401)
+        # if not request.user.is_authenticated:
+        #    return HttpResponse(status=401)
```

Comment:
We should be checking if the user is indeed authenticated.

Reply:
Sharing login status is not yet implemented. I will remove comments later.